Lib/gettext.py

```diff
@@ -491,11 +492,16 @@ def find(domain, localedir=None, languages=None, all=False):
         localedir = _default_localedir
     if languages is None:
         languages = []
-        for envar in ('LANGUAGE', 'LC_ALL', 'LC_MESSAGES', 'LANG'):
-            val = os.environ.get(envar)
-            if val:
-                languages = val.split(':')
-                break
+        if os.environ.get('LANGUAGE'):
+            languages = os.environ.get('LANGUAGE').split(',')
```

Comment:
Shouldn't this be `:` instead of `,`? Maybe I'd also add a test for when the env vars contain more than one language.